web/js/map-OpenLayers.js

```diff
@@ -333,23 +333,61 @@ var fixmystreet = fixmystreet || {};
 
     function onload() {
         if ( fixmystreet.area.length ) {
+            var extent = new OpenLayers.Bounds();
+            var lr = new OpenLayers.Geometry.LinearRing([
+                new OpenLayers.Geometry.Point(20E6,20E6),
+                new OpenLayers.Geometry.Point(10E6,20E6),
+                new OpenLayers.Geometry.Point(0,20E6),
+                new OpenLayers.Geometry.Point(-10E6,20E6),
+                new OpenLayers.Geometry.Point(-20E6,20E6),
+                new OpenLayers.Geometry.Point(-20E6,0),
+                new OpenLayers.Geometry.Point(-20E6,-20E6),
+                new OpenLayers.Geometry.Point(-10E6,-20E6),
+                new OpenLayers.Geometry.Point(0,-20E6),
+                new OpenLayers.Geometry.Point(10E6,-20E6),
+                new OpenLayers.Geometry.Point(20E6,-20E6),
+                new OpenLayers.Geometry.Point(20E6,0)
+            ]);
+            var loaded = 0;
+            var new_geometry = new OpenLayers.Geometry.Polygon(lr);
             for (var i=0; i<fixmystreet.area.length; i++) {
-                var area = new OpenLayers.Layer.Vector("KML", {
+                area = new OpenLayers.Layer.Vector("KML", {
```

Comment:
Doh, yes, I briefly tried it in a `$.each` instead of a for loop, so it was okay then, will fix!